server/config/config.go

```diff
@@ -650,6 +651,8 @@ type ScheduleConfig struct {
 	MaxMergeRegionKeys uint64 `toml:"max-merge-region-keys" json:"max-merge-region-keys"`
 	// SplitMergeInterval is the minimum interval time to permit merge after split.
 	SplitMergeInterval typeutil.Duration `toml:"split-merge-interval" json:"split-merge-interval"`
+	// SwitchWitnessInterval is the minimum interval time to permit to witness after promote to non-witness.
+	SwitchWitnessInterval typeutil.Duration `toml:"switch-witness-interval" json:"swtich-witness-interval"`
```

Comment:
Does it mean if a peer is promoted to non-witness, it cannot be changed back to witness unless spending `SwitchWitnessInterval`?

Reply:
Yes, as discussed in https://github.com/pingcap/kvproto/pull/999. If no TTL, it may schedule loop.